src/helpers/elm/ELMInfoCache.ts

```diff
@@ -58,7 +60,7 @@ export function retrieveELMInfo(
           if (msrPop?.criteria?.expression && obsrvPop.criteria?.expression) {
             const mainLib = elmJSONs.find(elm => elm.library.identifier.id === rootLibIdentifier.id);
             if (mainLib) {
-              const elmFunction = isEpisodeOfCareGroup(measure, group)
+              const elmFunction = MeasureBundleHelpers.isEpisodeOfCareGroup(measure, group)
```

Comment:
Just out of curiosity, why the change? If this is preferred maybe ignore my above comment and then change the code in `Execution.ts` to use this pattern, but up to you :)

Reply:
I made the change because it looks like we are importing `MeasureBundleHelpers` twice, once to import everything and then to also import `isEpisodeOfCareGroup`. Might change this to import just what we need